src/apps/mux/helpers/mux.mlp.token-helper.ts

```diff
@@ -54,7 +55,13 @@ const assetsTokensQuery = gql`
     }
   }
 `;
-const invalidAddress = '0x0000000000000000000000000000000000000000';
+
+const SUBGRAPH_URL: Partial<Record<Network, string>> = {
+  [Network.ARBITRUM_MAINNET]: 'https://api.thegraph.com/subgraphs/name/mux-world/mux-arb',
+  [Network.AVALANCHE_MAINNET]: 'https://api.thegraph.com/subgraphs/name/mux-world/mux-ava',
+  [Network.BINANCE_SMART_CHAIN_MAINNET]: 'https://api.thegraph.com/subgraphs/name/mux-world/mux-bsc',
+  [Network.FANTOM_OPERA_MAINNET]: 'https://api.thegraph.com/subgraphs/name/mux-world/mux-ftm',
+};
```

Comment:
When this is refactored to templates, prefer to set these as abstract properties.